climada/entity/impact_funcs/base.py

```diff
@@ -127,4 +127,65 @@ def check(self):
                            'origin of the intensity scale. In impact.calc '
                            'the impact is always null at intensity = 0.')
 
+    def set_step_ImpF(self, threshold, inten_min, inten_max, if_id=1):
+
+        """ Step function type impact function. Everything is destroyed above
+        threshold. Usefull for high resolution modelling.
```

Comment:
```suggestion
        threshold. Useful for high resolution modelling.
```

Reply:
Why is a step function useful for high resolution modeling?

Reply:
as high resolution modelling means "hit-or-miss" for many hazards (i.e. wildfire, landslides, flooding,...). for that step functions are quite useful (i.e. in our calibration for wildfire, our sigmoid function converged to a step on 1-km resolution)